spec/features/list_collections_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
```

Comment:
Why is this a feature spec and not a request spec? A request spec is for a single request, whereas a feature spec typically crosses a number of requests.